### ISSUE TITLE
problem verify signature

### ISSUE THREAD

**Issue by epninos** [micahflee/torbrowser-launcher#467]
while installing tor on fedora 31 and my xiaomi android cellphone was ridiculously easy,
and only took me just 1 minute,
installing it on ubuntu 18.04 is a real pain!

when I run the launcher
it down loads the bundle over again, but
it cannot verify the signature

error code: GENERIC VERIFY FAIL

I started the procedure 4 times
alas, to no avail

**Comment by ravishtiwari** [comment]
Yes, never had this issue earlier, tried using Binaries from Tor Project as well as *apt*. 
OS Details: 
```
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.4 LTS
Release:        18.04
Codename:       bionic
```

Getting signature error:
![Tor-Error](https://user-images.githubusercontent.com/1816459/81139656-20627580-8f84-11ea-907f-5aa8fb8a2b8b.png)

**Comment by CaracalOcelot** [comment]
+1 Exact same issue.

**Comment by Fyler74** [comment]
Same problem here.

**Comment by pastk** [comment]
I confirm same issue for me.

**Comment by pastk** [comment]
This [fix](https://github.com/micahflee/torbrowser-launcher/issues/458#issuecomment-619335312) solved it for me (Ubuntu 18.04) - the torbrowser-launcher package in 18.04 is old and probably doesn't work nicely with some packages that were updated like gpg.
